specs/linux/cpu_time.table

```diff
@@ -0,0 +1,16 @@
+table_name("cpu_time")
+description("Displays information from /proc/stat about the time the cpu cores spent in different parts of the system.")
+schema([
+    Column("core", INTEGER, "Name of the cpu (core)"),
+    Column("user", BIGINT, "Time spent in user mode"),
+    Column("nice", BIGINT, "Time spent in user mode with low priority (nice)"),
+    Column("system", BIGINT, "Time spent in system mode"),
+    Column("idle", BIGINT, "Time spent in the idle task"),
+    Column("iowait", BIGINT, "Since Linux 2.5.41 the time waiting for I/O to complete"),
```

Comment:
I'm not sure how helpful the `Since Linux X.Y.Z` preambles are, ideally we'll attempt to report the same types of column on OS X, FreeBSD, and one day Windows. Let's drop this detail and just document what someone should expect to find if the column is filled in.